### PR TITLE
Brid player support for carousel

### PR DESCRIPTION
**Pull request by grajzer**

### CODE REVIEW

**Review by alanorozco — COMMENTED** [review]
Would you mind adding a test for the required attribute error? There's a couple of similar tests:

https://github.com/ampproject/amphtml/blob/1e15159d609ecf1f9584191612e0f3e1d4a41bcc/extensions/amp-brid-player/0.1/test/test-amp-brid-player.js#L103-L112

File: extensions/amp-brid-player/validator-amp-brid-player.protoascii [hunk 1]
```diff
@@ -52,12 +52,17 @@ tags: {  # <amp-brid-player>
...
     value_regex: ".+"
   }
   attrs: {
     name: "data-video"
-    mandatory_oneof: "['data-outstream', 'data-playlist', 'data-video']"
+    mandatory_oneof: "['data-outstream', 'data-playlist', 'data-carousel', 'data-video']"
+    value_regex: "[0-9]+"
+  }
+  attrs: {
+    name: "data-carousel"
```
Comment (honeybadgerdontcare) [inline extensions/amp-brid-player/validator-amp-brid-player.protoascii:64]: please list in alphabetical order by attribute name.

**Review by honeybadgerdontcare — APPROVED** [review]
approved for validation after alphabetical change and adding a test per alan

**Review by alanorozco — CHANGES_REQUESTED** [review]
Requesting changes for test mentioned above.

**Comment by grajzer** [comment]
Thank you @alanorozco @honeybadgerdontcare , these changes are implemented.

**Review by alanorozco — APPROVED** [review]

**Pull request merged**